onomy/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "onomy",
+  "assets": [
+    {
+      "description": "The native token of Onomy Protocol",
+      "denom_units": [
+        {
+          "denom": "anom",
+          "exponent": 0
+        },
+        {
+          "denom": "nom",
+          "exponent": 18
+        }
+      ],
+      "base": "anom",
+      "name": "Nom",
+      "display": "nom",
+      "symbol": "NOM",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/onomy/images/nom.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/onomy/images/nom.svg"
```

Comment:
It looks to me that the svg version of the image doesn't have transparency in the corners. Could you please find one that does?

Reply:
Removed